umm-lib/project.clj

```diff
@@ -50,7 +50,9 @@
              :internal-repos {}
              :kaocha {:dependencies [[lambdaisland/kaocha "1.0.732"]
                                      [lambdaisland/kaocha-cloverage "1.0.75"]
-                                     [lambdaisland/kaocha-junit-xml "0.0.76"]]}}
+                                     [lambdaisland/kaocha-junit-xml "0.0.76"]
+                                     ;; ring is needed or this fails in sys int group3
```

Comment:
I'd like to see whats going on here.

Reply:
we can further investigate this in https://bugs.earthdata.nasa.gov/browse/CMR-10550